### DISCUSSION TITLE
Hello all, interested in contributing!

### DISCUSSION THREAD

**Discussion by rushowr** [freeplane/docs#144] [General]
Hello everyone from the Southside, Saint Louis, MO, USA! I'm S. "Ronin" McGowan, a man of many skills, master of none, but a dogged pursuer of knowledge and the sharing of such. That said, per the readme, I'm starting this thread to express interest in contributing to the documentation. 

I'm not sure what else is needed from me immediately, so I'll just wait to hear from the staff :) 
_edit_ I will add, I've created a fork of the documentation and am standing by for any instructions regarding specific tasks or anything else.

**Comment by macmarrum** [comment]
Hello S. "Ronin" McGowan from the Southside, Saint Louis, MO, USA.
Welcome to the Freeplane documentation project!

I'm guessing you're a Freeplane user and have already some experience with the tool. You probably had some "aha!" moments when discovering what and how about Freeplane. It might be a good point to start, i.e. to find places in the wiki pages and improve them based on your own experience (what would have helped you).

Another area in which help would be appreciated is [attic](https://docs.freeplane.org/attic/home.html). It contains draft documents, separated into

* `old-mediawiki-content`– MediaWiki pages (pre-2022 wiki) migrated to Markdown for GitHub,
* `new-wiki-initiative`– pages created around 2021 and later as an attempt to start using GitHub and Markdown for documentation.

The vision is to clean up `attic`.
Some content in `attic`, especially in `old-mediawiki-content`, might be outdated, some unconverted, some media files missing.
Therefore, the outcome to be achieved is to have each page reviewed/completed before incorporating it to `docs root` and/or removing it from `attic` as no longer relevant.

When a page is incorporated to `docs root` and/or removed from `attic`, all existing `attic` pages linking to it should be updated.
Relative links should be used.

You can also have a look at 
* https://github.com/freeplane/docs/discussions/120

**Reply by rushowr** [reply]
Ahoy @macmarrum! Happy to be here, and thank you for the welcome and the guidance. I am indeed a Freeplane user, although I haven't been recently, but only due to lack of need at the present moment :). I'm going to enjoy getting back into Freeplane and contributing what I can. 
I will dig in and of course follow the additional guidance of the linked document.
